libobs/pkgconfig/libobs.pc.in

```diff
@@ -6,5 +6,5 @@ includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@/obs
 Name: libobs
 Description: OBS Studio Library
 Version: @OBS_VERSION_CANONICAL@
-Cflags: -I${includedir} @_TARGET_DEFINITIONS@ @_TARGET_OPTIONS@ @_LINKED_DEFINITIONS@
-Libs: -L${libdir} @_LINKED_LIBRARIES@
+Cflags: -I${includedir} @_TARGET_DEFINITIONS@ @_TARGET_OPTIONS@ @_LINKED_DEFINITIONS@ @_LINKED_OPTIONS@
+Libs: -L${libdir} -llibobs @_LINKED_LIBRARIES@
```

Comment:
Sorry, I don't think `-llibobs` is correct. `Libs` is forwarded to the linker and, according to [ld man pages](https://man7.org/linux/man-pages/man1/ld.1.html), specifically the `--library` secion, it must be either `:filename`, or a `namespec`. The latter is searched as `lib[namespec] .so` first, and `lib[namespec].a` second, on ELF-based systems such as Linux and BSDs. It is preferred to use name specs instead of filenames.

So, in the all, all that was to say `-lobs` is the appropriate declaration in pkg-config.